docs/tutorials/notebooks/tracking_progress.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>Consider showing <code>ot.converged</code> / <code>ot.n_iters</code>  / <code>ot.reg_ot_cost</code> so that it doesn't feel so empty (maybe even show the coupling, will leave it up to you).</p><p>The downside of the <code>n_iters </code> is that it's not implemented for LR case...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I included this and rewrote this section for clarity.</p>